compiler/exe4j/compiled-with-exe4j.yml

```diff
@@ -0,0 +1,19 @@
+rule:
+  meta:
+    name: compiled with exe4j
+    namespace: compiler/exe4j
+    author: johnk3r
+    scope: file
+    att&ck:
+      - Defense Evasion::Obfuscated Files or Information::Compile After Delivery [T1027.004]
```

Comment:
lets remove this because exe4j is not specifically for obfuscation

Reply:
Ok. I took as reference other rules in the namespace compiler ...